### DISCUSSION TITLE
"Fzflua files" isn't giving me any files

### DISCUSSION THREAD

**Discussion by utrumsit** [ibhagwan/fzf-lua#1573] [Q&A] [answered]
Lazyvim recently switched over to fvf-lua, so I've been trying it. When I attempt to search files I just get an empty window. I tried it by downloading the script on the website and running it in a clean nvim, and it still doesn't work. "Fzf-lua oldfiles" works as expected. 

Any ideas how I can get this to work? Linux mint, I just made sure I had 0.57 fzf. Fzf works fine from the command line.

**Comment by ibhagwan** [comment]
So you have `fd` or `rg` installed? Can you run `:FzfLua files debug=true` and post the sceeenwhot with the debug line after the interface os open? It should show what underlying command is running.

**Comment by utrumsit** [comment, marked as answer]
Ah, I figured it out. Apologies! I thought `fd` was installed because I could type "fd" and it would find files. But I needed to remove fdclone, install fd-find, and then link "fd" to fd-find, as described here: [https://github.com/sharkdp/fd](https://github.com/sharkdp/fd)

Now it works as expected. 

``
